### PR TITLE
add svg pagination

### PR DESCRIPTION
**Pull request by yyrcat**

### CODE REVIEW

File: assets/scripts/evolution/svg_pagination.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export default function() {
+	(function() {
+		var elems = document.querySelectorAll('ul.evo_c-svg-dot--default li'),
+		makeActive;
+
+		makeActive = function() {
```
Comment (jmatembu) [inline assets/scripts/evolution/svg_pagination.js:6]: @yyrcat 

This code will work perfectly when `ul.evo_c-svg-dot--default li` exists in the markup. In cases where those elements are not present, the code will break.. The solution here is to check if `elems` exist. See detailed instructions here on writing safe javascript code: https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#writing-safe-javascript-code

Comment (yyrcat) [reply assets/scripts/evolution/svg_pagination.js:6]: Hello @jmatembu , after I finished the modification, so I just need to push again or create another pull request? Thanks!

Comment (jmatembu) [reply assets/scripts/evolution/svg_pagination.js:6]: Yeah sure @yyrcat, just push.

Comment (yyrcat) [reply assets/scripts/evolution/svg_pagination.js:6]: hello @jmatembu , I met a problem, when I push, it says my local  c-svg-pagination is behind the remote c-svg-pagination. So should I pull the remote branch firstly? Thanks!

**Pull request closed without merging**